_setup-python/action.yml

```
@@ -47,8 +47,9 @@ inputs:
         When uv cache is pruned, pre-built wheels are removed but wheels built from source
         are retained.
 
-    required: true
+    required: false
```

```suggestion
    required: true
```
By convention all inputs of "private" action are required